File: Program.cs/Number16.cs
// C# program to illustrate the 
// String.IndexOf(string s1) method 
using System; 
namespace ConsoleApplication4 { 

class Geeks { 

	// Main Method 
	static void Main(string[] args) 
	{ 

		string str = "Jeruk Apel Anggur Pepaya Pisang Kiwi Markisa"; 

		int i = str.IndexOf("Jeruk"); 

		// As this string is present in the 
		// main string then it will obviously 
		// output the value as 17 
		Console.WriteLine("First value Index of 'Jeruk' is " + i); 

		// now the following string is not present 
		// So as per the rules, it will return -1 
		int i1 = str.IndexOf("Pisang"); 

		// As this string is present in 
		// the main string then it will 
		// obviously output the value as -1 
		Console.WriteLine("First value Index of 'Pisang' is " + i1); 
	} 
} 
} 

File: Program.cs/Number16.cpp
#include <iostream> 
#include <string> 

using namespace std; 

int main() 
{ 
	string str = "jeruk apel anggur pepaya pisang kiwi markisa"; 
	string str1 = "anggur"; 

	// Find first occurrence of "geeks" 
	size_t found = str.find(str1); 
	if (found != string::npos) 
		cout << "First occurrence is " << found << endl; 

	char arr[] = "anggur"; 
	found = str.find(arr, found+1); 
	if (found != string::npos) 
		cout << "Next occurrence is " << found << endl; 

	return 0; 
} 
